src/tagmanager/tm_parser.c

```diff
@@ -557,6 +562,7 @@ static TMParserMapEntry map_FREEBASIC[] = {
 	{'t', tm_tag_struct_t},     // type
 	{'v', tm_tag_variable_t},   // variable
 	{'g', tm_tag_externvar_t},  // enum
+	{'n', tm_tag_undef_t},      // namespace
```

Comment:
No clue about freebasic, but wouldn't it be worth including?  kinds are already silly, so the fact `tm_tag_namespace_t` is already in use is not really a reason not to add it, is it?

Reply:
Yes, these might be missing in the hierarchy. Added them in the latest commit including the unit test I copied from universal-ctags.